### PR TITLE
feat: ocular-tsify using ts-smoosh

### PR DESCRIPTION
**Pull request by ilijapuaca**
Integrates `ts-smoosh` work that @heshan0131 has done and exposes it through `ocular-tsify` command

### CODE REVIEW

File: modules/dev-tools/src/ts-smoosh/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+## TS Smoosh
```
Comment (ibgreen) [inline modules/dev-tools/src/ts-smoosh/README.md:1]: Move to docs?

File: modules/dev-tools/src/ts-smoosh/log.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const Colors = {
```
Comment (ibgreen) [inline modules/dev-tools/src/ts-smoosh/log.js:1]: We are using probe.gl logging which has similar features, maybe leave a comment?

**Review by ibgreen — APPROVED** [review]

**Comment by ibgreen** [comment]
@btford FYI your ts-smooch will live forever in ocular.

File: modules/dev-tools/src/ts-smoosh/README.md (hunk 1 above)
Comment (ilijapuaca) [reply modules/dev-tools/src/ts-smoosh/README.md:1]: Some of the implementation details like running tests aren't relevant outside the context of this source. Added a new doc `ocular-tsify.md` alongside other commands that contains a subset that is relevant to the end user

**Pull request merged**